index.js

```diff
@@ -32,20 +32,20 @@ const slowDown = PCancelable.fn(async ({config, convert, exportOptions, inputPat
 });
 
 const slowDownService = {
-	title: 'Slow down',
+	title: 'Slow Down',
```

Comment:
We should document in the docs that all titles should be titlecase. Including the main title and config property titles.